src/pages/ProfilePage.js

```diff
@@ -0,0 +1,345 @@
+    } else {
+      setStorageCatCutieName(event.target.value);
+    }
+  };
+
+  const [catAge, setStorageCatAge] = useLocalStorage('catAge', 0);
+
+  const [editingCatAgeValue, setEditingCatAgeValue] = useState(catAge);
+
+  const onBlurCatAge = event => {
+    if (event.target.value.trim() === '') {
+      setEditingCutieNameValue(catAge);
+    } else {
+      setStorageCatAge(event.target.value);
+    }
+  };
+
+  const [note, setStorageNote] = useLocalStorage('note', '...');
+
+  const [editingNoteValue, setEditingNoteValue] = useState(note);
+
+  const onBlurNote = event => {
+    if (event.target.value.trim() === '') {
+      setEditingNoteValue(note);
+    } else {
+      setStorageNote(event.target.value);
+    }
+  };
+
+  const onKeyDown = event => {
+    if (event.key === 'Enter' || event.key === 'Escape') {
+      event.target.blur();
+    }
+  };
+
+  return (
+    <FormPageStyle>
+      <Header>
+        <h1>Mein Katzenprofil</h1>
+        <ExitButton />
+      </Header>
+      <main>
+        <ImgContainer min-width="280px" min-height="165px">
+          <img src={image} alt="" width="280px" height="165px" />
+          <RemoveButton onClick={onImageRemove} data-testid="buttonRemove">
+            <span className="sr-only">Image remove button</span>
+            <img src={iconbin} alt="icon rotate arrow to the left" />
+          </RemoveButton>
+          <input
+            data-testid="inputImgUpload"
+            id="imgUpload"
+            type="file"
+            onChange={upload}
+            hidden
+          />
+          <UploadButton htmlFor="imgUpload" data-testid="inputLabelImgUpload">
+            <span className="sr-only">Image upload and change</span>
+            <img src={iconrotate} alt="bin icon" />
+          </UploadButton>
+        </ImgContainer>
+
+        <FormContainer>
+          <div>
+            <img src={iconpencil} width="20px" height="20px" alt="pen" />
+            <InputTextStyled
+              data-testid="inputCatName"
+              type="text"
+              id="catname"
+              value={editingNameValue}
+              onChange={event => setEditingNameValue(event.target.value)}
+              onKeyDown={onKeyDown}
+              onBlur={onBlurName}
+              maxLength={30}
+            />
+            <label className="sr-only" htmlFor="catname">
+              Katzenname
+            </label>
+          </div>
+          <div>
+            <img src={iconpencil} width="20px" height="20px" alt="pen" />
+            <InputTextStyled
+              type="text"
+              id="petname"
+              data-testid="inputPetName"
+              value={editingCutieNameValue}
+              onChange={event => setEditingCutieNameValue(event.target.value)}
+              onKeyDown={onKeyDown}
+              onBlur={onBlurCutieName}
+              maxLength={30}
+            />
+            <label htmlFor="petName" className="sr-only">
+              Kosename des Katze
```

Comment:
Small grammer error. You probably meant `Kosename der Katze`

Reply:
many thanks for finding this 😸